app/org/sagebionetworks/bridge/dynamodb/StringValueStringKeyMapMarshaller.java

```diff
@@ -0,0 +1,14 @@
+package org.sagebionetworks.bridge.dynamodb;
+
+import java.util.Map;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+
+public class StringValueStringKeyMapMarshaller extends StringKeyMapMarshaller<String> {
```

Comment:
Dynamo handles Maps just fine. Is there any particular reason we need to marshal to/from JSON?

Reply:
Switched to @DynamoDBTypeConvertedJson.